.github/workflows/ci.yml

```
@@ -0,0 +1,108 @@
+## (5b) Pin to commits, not tags
+
+name: CI Build and Check
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  IMAGE: rancher/kube-api-auth
+
+permissions:
+  contents: read
+  security-events: write # upload Trivy Sarif results
+
+jobs:
+  linter:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Get Sources
+      uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v5
+      with:
+        go-version-file: 'go.mod'
+        cache: false
+
+    - name: Validate the sources
+      run: |
+        ./scripts/validate
+        # Except for linting. That is done through the proper GH action
+
+    - name: Lint sources
+      uses: golangci/golangci-lint-action@v4
+
+    - name: Test
+      run: ./scripts/test
+
+  build:
+    runs-on: ubuntu-latest
+    needs:
+      - linter
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - linux
+        arch:
+          - amd64
+          - arm64
+    steps:
+    - name: Prepare Matrix Instance
+      run: |
+        echo "GOOS=${{ matrix.os }}"
+        echo "GOOS=${{ matrix.os }}"     >> $GITHUB_ENV
+        echo "GOARCH=${{ matrix.arch }}"
+        echo "GOARCH=${{ matrix.arch }}" >> $GITHUB_ENV
+        echo "ARCH=${{ matrix.arch }}"
+        echo "ARCH=${{ matrix.arch }}"   >> $GITHUB_ENV
+
+    - name: Get Sources
+      uses: actions/checkout@v3
+
+    - name: Determine Tag
+      id: get-TAG
+      run: |
+        export TAG=g$(git describe --always)
```

I don't think the `g` is intentional
```suggestion
        export TAG=$(git describe --always)
```

---

Actually it was intentional, `g` for `here comes a git commit hash`.

---

Changed with commit dd3c282ee030db5090de1eeda91109eba4f1786a